napari/_vispy/vispy_image_layer.py

```diff
@@ -95,21 +86,21 @@ def _on_data_change(self, event=None):
                 self.node.set_data(data, clim=self.layer.contrast_limits)
         self.node.update()
 
-    def _on_interpolation_change(self, event=None):
+    def _on_interpolation_change(self, interpolation):
```

Comment:
while we're doing this, it might be nice to standardize even the argument naming.  so `interpolation` here (and all the other setters) could just be `value`